app/controllers/foreman_tasks/tasks_controller.rb

```diff
@@ -98,7 +98,7 @@ def resource_class
     end
 
     def auto_complete_controller_name
-      '/foreman_tasks/tasks'
```

Comment:
This code has actually been added in #375 to workaround regression introduced in theforeman/foreman#5771. It seems that it has not fixed the issue properly.

In order for this fix to work, we need also theforeman/foreman#6460.

After that is merged, we can just remove the auto_complete_controller_name method complete from here, so that it would work as before #375.

Reply:
The foreman PR has just been merge. Mind updating the PR to just remove this method completely? That should fix the whole problem.